trade-finance-manager-ui/server/controllers/case/underwriting/index.test.js

```diff
@@ -53,18 +55,27 @@ describe('GET getUnderwriterPage', () => {
         user: MOCKS.MOCK_USER_UNDERWRITER_MANAGER,
         userCanEdit: true,
       },
+      successMessage: mockSuccessBannerMessage,
```

Comment:
I have a lot of thoughts about these tests, but it isnt a discussion for this PR. Good job managing to work with them though.